### PR TITLE
dosbox-staging 0.75.0 (new formula)

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Previous attempt: #54659

Based on [dosbox](https://github.com/Homebrew/homebrew-core/blob/master/Formula/dosbox.rb) formula

### CODE REVIEW

**Comment by SeekingMeaning** [comment]
Todo: add `depends_on "pkg-config" => :build`

File: Formula/dosbox-staging.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ]
+
+    system "./autogen.sh"
+    system "./configure", *args
+    system "make", "install"
+    mv bin/"dosbox", bin/"dosbox-staging"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/dosbox-staging -version")
```
Comment (claui) [inline Formula/dosbox-staging.rb:32]: I still feel https://github.com/Homebrew/homebrew-core/pull/54659#discussion_r424669240 is relevant here.

Is there no way to exercise even very basic functionality?

Comment (SeekingMeaning) [reply Formula/dosbox-staging.rb:32]: Hmm there's a `-printconf` flag that outputs the path to a configuration file named `dosbox-staging.conf`

Other than that there only seems to be a `-editconf` flag that opens the config file in a text editor

Source: https://github.com/dosbox-staging/dosbox-staging/blob/master/src/gui/sdlmain.cpp

Comment (claui) [reply Formula/dosbox-staging.rb:32]: Nice!
That `-printconf` thing sounds just right.

Comment (SeekingMeaning) [reply Formula/dosbox-staging.rb:32]: Done ✅ Took me awhile to realize that `testpath` could be treated like `HOME`

**Review by SMillerDev — APPROVED** [review]
Thanks @SeekingMeaning ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**